elasticsearch-core/src/test/scala/com/sumologic/elasticsearch/restlastic/RestlasticSearchClientTest.scala

```diff
@@ -1209,7 +1209,8 @@ trait RestlasticSearchClientTest {
       val termQuery = TermQuery("text7", "here7")
 
       val delFut = restClient.deleteByQuery(index, tpe, new QueryRoot(termQuery), true)
```

Comment:
Please also add a test case with `val delFut = restClient.deleteByQuery(index, tpe, new QueryRoot(termQuery), false)`

In this test case, check only the response, not the count after sending delete request, since it's asynchronous.

Reply:
Done